src/server/services/afis/afis-facturen.ts

```diff
@@ -260,15 +260,43 @@ function transformFacturen(
 ): AfisFacturenResponse {
   const feedProperties = getFeedEntryProperties(responseData);
   const count = responseData?.feed?.count ?? feedProperties.length;
-  const facturenTransformed = feedProperties.map((invoiceProperties) => {
-    return transformFactuur(invoiceProperties, sessionID, deelbetalingen);
-  });
+  const facturenTransformed = feedProperties
+    .filter((invoiceProperties) => {
+      return shouldFilterOutFactuur(invoiceProperties.PostingDate);
+    })
+    .map((invoiceProperties) => {
+      return transformFactuur(invoiceProperties, sessionID, deelbetalingen);
+    });
   return {
     count,
     facturen: facturenTransformed,
   };
 }
 
+function shouldFilterOutFactuur(postingDate: string): boolean {
+  const postingDateAsDate = new Date(postingDate);
+  const now = new Date();
+
+  if (postingDateAsDate.getTime() > now.getTime()) {
+    captureMessage('postingDate of invoice is in the future', {
+      severity: 'error',
+    });
```

Comment:
Is dit echt nodig?

Reply:
Denk het niet eigenlijk, ik haal het weg.